### PR TITLE
Create duesseldorf.json

### PR DESCRIPTION
**Pull request by ufh**
Softwerkskammer Düsseldorf to host the GDCR2019.

### CODE REVIEW

File: _data/events_gdcr2019/duesseldorf.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      "name": "Birgit Kratz",
+      "url": "https://www.xing.com/profile/Birgit_Kratz2"
+    },
+    {
+      "name": "Tim Riemer",
+      "url": "https://twitter.com/zordan_f"
+    },
+    {
+      "name": "Michael Arends",
+      "url": "https://corporate.vorwerk.de/home/"
```
Comment (rachelcarmena) [inline _data/events_gdcr2019/duesseldorf.json]: This is the same link of sponsor. Please, if Michael Arends doesn't have a personal page (Twitter, Linkedin, Xing, ...), remove `url` property in this case. Thanks in advance!

Comment (ufh) [reply _data/events_gdcr2019/duesseldorf.json]: Hi Rachel, thanks for the hint. I removed the url and created a new pull request:
https://github.com/coderetreat/coderetreat.org/pull/556

Comment (rachelcarmena) [reply _data/events_gdcr2019/duesseldorf.json]: Great!! Thanks @ufh :tada:

**Comment by rachelcarmena** [comment]
Replaced by #556

**Pull request closed without merging**